RELEASE.md

```diff
@@ -0,0 +1,33 @@
+RELEASE_TYPE: minor
+
+This release changes how some of the SQS helpers work.
+
+*   `withLocalSqsQueueAndDlq` and `withLocalSqsQueueAndDlqAndTimeout` are both renamed to `withLocalSqsQueuePair()`, which takes an optional `visibilityTimeout` argument.
+
+    The DLQ name will also be the name of the main queue suffixed with `-dlq` for easier debugging.
+
+*   `withLocalSqsQueue` now takes an optional `visibilityTimeout` argument.
+
+    If you are using it without arguments, e.g.:
+
+    ```scala
+    withLocalSqsQueue { queue =>
+      // do stuff with queue
+    }
+    ```
+
+    you need to add empty parentheses:
+
+    ```scala
+    withLocalSqsQueue() { queue =>
```

Comment:
A bit late now but isn't this a breaking (ie `major`) change? Could you either add an overload so it could be used without parens or do something like `withLocalSqsQueueWith(...)` and `withLocalSqsQueue = withLocalSqsQueueWith()` as done in the `*Generators` helpers?

Reply:
It is, and also one I decided to push through anyway. We’ve never been great about semver, and it’s a pretty small fix that I can roll out quickly.